### ISSUE TITLE
Addison disease and chronic primary adrenal insufficiency

### ISSUE THREAD

**Issue by maglott** [monarch-initiative/mondo#1218]
MONDO_0000004, MONDO_0015129

Addison disease has an exact synonym of primary adrenocortical insufficiency
chronic primary adrenal insufficiency has an exact synonym of Addison disease and a definition that says it is a disorder of the adrenal cortex,

HPO equates https://hpo.jax.org/app/browse/term/HP:0008207 primary adrenal insufficiency with Addison.  

If it is critical to distinguish among primary and secondary adrenocortical insufficiency and Addison,  can the synonyms and definitions be clarified?

**Comment by nicolevasilevsky** [comment]
Here is a table showing Donna's comments above:

ID | label | syn | syn xref
-- | -- | -- | --
MONDO_0000004 | 'adrenocortical insufficiency'
MONDO_0015129 | 'chronic primary adrenal insufficiency' | Addison disease | NCIT:C26689
MONDO_0009410 | 'Addison disease' | primary adrenocortical insufficiency | DOID:13774

Seems like we could merge 'chronic primary adrenal insufficiency' and 'Addison disease'?

@pnrobinson can you help? I'm not an expert in this area.

**Comment by nicolevasilevsky** [comment]
Per Ada - yes merge

**Comment by nicolevasilevsky** [comment]
![issue-1218](https://user-images.githubusercontent.com/6722114/145281825-94e175ef-30ec-478a-8624-7042cd6e9018.png)

I will merge MONDO_0009410'Addison disease' into MONDO_0015129 'chronic primary adrenal insufficiency' but I think we need a new class for Autoimmune Primary Adrenal Insufficiency that is equiv to Orphanet and NCIT.

To do:
- [ ] MONDO_0009410'Addison disease' into MONDO_0015129 'chronic primary adrenal insufficiency'
- [ ] create new class Autoimmune Primary Adrenal Insufficiency that is equiv to Orphanet and NCIT (child of 'acquired chronic primary adrenal insufficiency')
